pkg/userdata/centos/userdata.go

```diff
@@ -259,7 +244,12 @@ write_files:
     [Service]
     Type=oneshot
     RemainAfterExit=true
-    ExecStart=/usr/local/bin/supervise.sh /usr/local/bin/setup
+    ExecStart=/opt/bin/supervise.sh /opt/bin/setup
+
+- path: "/etc/profile.d/opt-bin-path.sh"
```

Comment:
This will be sourced and should not be executable, hence have `permissions: "0644"`

Reply:
done